src/c_kzg_4844.c

```diff
@@ -1367,9 +1397,9 @@ static C_KZG_RET verify_kzg_proof_batch(
  * @param[in]  n                 The number of blobs/commitments/proofs
  * @param[in]  s                 The trusted setup
  */
-C_KZG_RET VERIFY_BLOB_KZG_PROOF_BATCH(
+C_KZG_RET verify_blob_kzg_proof_batch(
     bool *ok,
-    const Blob *blobs,
+    const uint8_t *blobs,
```

Comment:
Why not `**blobs`? I think `**blobs` implies an array of blobs.

Reply:
That would be an array of pointers. We expect a flattened, contiguous array of the blobs' bytes.

Reply:
I would prefer to explicitly clarify that in the doc. Currently, it says that "all input arrays contain n elements", which does not match the "flattened array of bytes" view.

Reply:
Hmm okay. Thoughts @asn-d6? I'd like to point out that `commitments_bytes` and `proofs_bytes` behave the same way. We're passing all of the bytes, not pointers to each commitment/proof. And under the hood, the blobs argument is passed to this function the same as it was before.

Reply:
I'm not opposed to changing this btw! It would remove the need to flatten the blobs in the bindings. This would be faster and less complicated. But it would require more changes to the bindings.

Reply:
I think this is a good point raised by @ppopth and @GottfriedHerold . IMO we should clarify the documentation of the function and not change its internals.

Reply:
Okay cool. Fixed in [e7d980e](https://github.com/ethereum/c-kzg-4844/pull/352/commits/e7d980e9d4bd15595706d6e1cc45445bbb7f6f23). Just a remark, but I think that's enough.